MinecraftClient/ChatBot.cs

```diff
@@ -878,48 +899,48 @@ protected bool InteractEntity(int EntityID, int type)
         }
 
         /// <summary>
-        /// Give Creative Mode items into regular/survival Player Inventory
+        /// Interact with an entity
         /// </summary>
-        /// <remarks>(obviously) requires to be in creative mode</remarks>
-        /// <param name="slot">Destination inventory slot</param>
-        /// <param name="itemType">Item type</param>
-        /// <param name="count">Item count</param>
-        /// <returns>TRUE if item given successfully</returns>
-        protected bool CreativeGive(int slot, ItemType itemType, int count)
+        /// <param name="slot"></param>
+        /// <param name="ItemType"></param>
+        /// <param name="count"></param>
+        /// <returns></returns>
+        protected bool CreativeInventoryGive(int slot, ItemType ItemType, int count)
         {
-            return Handler.DoCreativeGive(slot, itemType, count);
+            return Handler.DoCreativeGive(slot, ItemType, count);
         }
 
         /// <summary>
-        /// Plays animation (Player arm swing)
+        /// Send animation
         /// </summary>
-        /// <param name="animation">0 for left arm, 1 for right arm</param>
-        /// <returns>TRUE if animation successfully done</returns>
+        /// <param name="animation"> 0 or 1</param>
```

Comment:
Seems like your commits are not applied to the latest source tree, this pull request reverts some commits added since your last pull request. You may need to sync your fork, or fork again and reapply your changes. In any case, backup your project files in case you need to manually reinsert your changes. 😉